Makefile.package

```diff
@@ -105,10 +108,32 @@ help:
 	echo "  PACKAGE_VERSION: ${PACKAGE_VERSION}"
 	echo "  PACKAGE_CHANNEL: ${PACKAGE_CHANNEL}"
 	echo "  PACKAGE_FQN:     ${PACKAGE_FQN}"
+	echo "  PACKAGE_ID:      ${PACKAGE_ID}"
 	echo "  PACKAGE_DIR:     ${PACKAGE_DIR}"
+	echo "  PACKAGE_DATE:    ${PACKAGE_DATE}"
 	echo "  GIT_COMMIT_HASH: ${GIT_COMMIT_HASH}"
 	echo
 
+status:
+	# Show the status of each package in the cloe workspace:
+	# - Is it editable / built / packaged?
+	# - Is it available in debug mode?
+	# - Is it identical to that on the remote?
+	# - Is it dirty?
+	# - Are there changes from the package?
+	# - Is cloe-launch ready to use?
+	#
+	_editable=$$(conan editable list | grep "${PACKAGE_FQN}"); \
+	if [ -z "$${_editable}" ]; then \
+		if [ -n "${PACKAGE_DATE}" ] && [ -z "$$(find -type f -newermt "${PACKAGE_DATE}")" ]; then \
```

Comment:
You have some very valuable points here.
- I didn't want to do an export because that would modify the cache, and I don't want to do that for an operation that should be read-only.
- I want to let the user know that the package in the cache is *different* to the current source, so if I use the git log it will show the package as up-to-date if I ever switch to a branch that is older than the one that created the last build.
- Of course this also means if you checkout A, build, checkout B, checkout A -> make status will show in the worst case that everything as outdated

I'm going to leave this as is, but feel free to submit a PR improving it.